packages/assets/icons/icon-chevron-right.svg

```diff
@@ -1,4 +1,4 @@
 <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
   <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"/>
```

Comment:
Need to close the `<path>` with `</path>` for consistency and as can been seen in the templates.

Reply:
Resolved